apps/omg_watcher/lib/omg_watcher/exit_processor/core.ex

```diff
@@ -627,6 +626,6 @@ defmodule OMG.Watcher.ExitProcessor.Core do
     address != @zero_address
   end
 
-  defp sla_margin_safe?(exit_processor_sla_margin, min_exit_period_seconds, ethereum_block_time_seconds),
-    do: exit_processor_sla_margin * ethereum_block_time_seconds < min_exit_period_seconds
+  defp sla_seconds_safe?(exit_processor_sla_seconds, min_exit_period_seconds),
+    do: exit_processor_sla_seconds < min_exit_period_seconds
```

Comment:
```suggestion
  defp sla_seconds_safe?(exit_processor_sla_seconds, min_exit_period_seconds) do
    exit_processor_sla_seconds < min_exit_period_seconds
  end
```

Reply:
the function spreads over multiple lines, let's allow it to do so.

Reply:
👍